### ISSUE TITLE
对于不同采样率 (sampling rate / CS ratio) /不同测量条件 (noise / mask) 的 hyper-parameter 建议是否可以补充？

### ISSUE THREAD

**Issue by nleowyaahusyloauyshaysx-prog** [myheuf/TransCS#7]
由于不同采样率 /测量条件对重建任务影响较大。在仓库中，训练与测试示例仅给出一个采样率 (例如 rate=0.1) 和标准 datasets。 若用户希望使用其他采样率 /mask策略 / 自定义数据，则缺乏经验配置参考。

建议作者分享或补充一些经验值 (learning rate, epoch 数, batch size, mask / sampling matrix 初始化方式等) —— 或者提供 config file template —— 帮助用户在不同设置下复现实验或迁移任务。
